src/test/resources/projects/build-it/archetype.properties

```diff
@@ -25,4 +25,5 @@ logback-version=${logback-version}
 maven-compiler-plugin-version=3.8.1
 maven-resources-plugin-version=${maven-resources-plugin-version}
 spring-boot-version=${spring-boot-version}
-camel-version=3.14.0
+camel-version=3.20.1
```

Comment:
is it needed?

Reply:
AFAIK those properties are useful for default values definition in combination with archetype-metadata above. I.e.: you can run:
`mvn archetype:generate -DarchetypeGroupId=software.tnb -DarchetypeArtifactId=camel3-archetype-spring-boot -DarchetypeVersion=0.1.4 -DgroupId=test.group -DartifactId=build-it -Dversion=0.0.1 -Dpackage=org.apache.camel.archetypes.archetypeIT -Dmaven-compiler-plugin-version=3.8.1 -B`